examples/multi_deployment/main.tf

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  credentials_file_path = var.credentials_path
+}
+/*****************************************
+  Folder Structure
+ *****************************************/
+module "migration-folders" {
+  source  = "terraform-google-modules/folders/google"
+  version = "~> 2.0"
+  # parent = "organizations/${var.organization_id}"
+  parent            = "folders/619261348000"
+  names             = ["core-project", "velo", "prod", "stg", "test"]
+  set_roles         = true
+  per_folder_admins = var.per_folder_admins
+  all_folder_admins = var.all_folder_admins
+}
+
+/*****************************************
+  Core Project
+ *****************************************/
+
+module "core-project" {
```

Comment:
What's this used for?

Reply:
I named this wrong. It is the Shared VPC Host Project.
https://github.com/terraform-google-modules/terraform-google-migrate/blob/dev/images/cft-velo-multi.png